### PR TITLE
fixed

### PR DESCRIPTION
**Pull request by LordStephen77**

### CODE REVIEW

**Comment by LordStephen77** [comment]
Travis is configured badly I think. Travis and Gradle are a bunch of problems ;) A greeting

File: src/simplejavatexteditor/SupportedKeywords.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
  */
 public class SupportedKeywords {
 
-    private String[] supportedLangage = {".cpp",".java"};
+    private String[] supportedLanguage = {".cpp",".java"};
```
Comment (pH-7) [inline src/simplejavatexteditor/SupportedKeywords.java:15]: Thanks for fixing the typo! You could also change it to plural to be `supportedLanguages` to be clearer.

File: src/simplejavatexteditor/SupportedKeywords.java [hunk 2]
```diff
@@ -33,8 +33,8 @@
                 "template", "typename", "class", "friend", "private", "this", "using", "const_cast",
                 "inline", "public", "throw", "virtual", "delete", "mutable", "protected", "true", "wchar_t" };
 
-    public String[] getSupportedLangage() {
-        return supportedLangage;
+    public String[] getsupportedLanguage() {
```
Comment (pH-7) [inline src/simplejavatexteditor/SupportedKeywords.java:36]: As long as you are here, you could also change its name to be plural ;)
`public String[] getsupportedLanguages() {`

**Review by pH-7 — CHANGES_REQUESTED** [review]

**Comment by pH-7** [comment]
@LordStephen77 Let me know if you had the change to see my comments above  ;)

File: src/simplejavatexteditor/SupportedKeywords.java (hunk 1 above)
Comment (pH-7) [reply src/simplejavatexteditor/SupportedKeywords.java:15]: @LordStephen77 Did you get a chance to look at my comment?

File: src/simplejavatexteditor/SupportedKeywords.java (hunk 2 above)
Comment (pH-7) [reply src/simplejavatexteditor/SupportedKeywords.java:36]: @LordStephen77 Did you get a chance to look at my comment

**Comment by pH-7** [comment]
Because I didn't get any answers from you, I have to close this one in order to keep the workflow cleanned.

File: .gitignore [hunk 3]
```diff
@@ -6,3 +6,4 @@
 # Mac OS System
 .DS_Store*
 ._*
+/bin/
```
Comment (pH-7) [inline .gitignore:9]: I think this one isn't necessary. Could be removed ;)

**Pull request closed without merging**